addons/binding/org.openhab.binding.rfxcom/ESH-INF/thing/channels.xml

```diff
@@ -170,4 +170,24 @@
 		<state min="0" max="360" step="1" readOnly="true"></state>
 	</channel-type>
 
+	<channel-type id="scene">
+		<item-type>Number</item-type>
+		<label>Scene</label>
+		<description>Scene channel</description>
+		<state readOnly="false">
+			<options>
+				<option value="0">Scene 1</option>
```

Comment:
I would prefer to keep the abstraction out of the binding itself. I feel like the binding is purely for mapping the RFXCOM to items, adding abstraction to that would make the code more complicated than it needs to be. The abstraction is possible, and more flexible, with rules. @kaikreuzer this might be more of an architectural question. How do you feel about putting this kind of functionality into the binding rather than relying on rules?

I do get your point about there not being an ON, but using a scene to replace ON only works if the scene is correctly configured anyway, so it is potentially more confusing to users that way. Maybe what we should do is add a TOGGLE command to the list, TOGGLE(0x01, LIVOLO_APPLIANCE), so that it is clear than the LIVOLO has no "ON" command.